src/control/cmd/dmg/system.go

```diff
@@ -119,7 +119,7 @@ func (cmd *systemQueryCmd) Execute(_ []string) (errOut error) {
 	}
 	req := new(control.SystemQueryReq)
 	req.Hosts.ReplaceSet(hostSet)
```

Comment:
Agreed, just wasn't sure it should be done as part of this PR. If that doesn't bother anyone, I'll go ahead and do it now. The reason I made the change in the new types is because the "Set" part of it exposes implementation details and isn't correct/helpful IMO.